### DISCUSSION TITLE
Redis Search

### DISCUSSION THREAD

**Discussion by AruneshSingh** [superlinked/VectorHub#81] [VDB Comparison]
Hey everyone, this is the start of discussions related to [Redis Search](https://redis.com/solutions/use-cases/vector-database/) and their attributes as compared in the table [here](https://vdbs.superlinked.com). 

To add to the discussion please check if there's already a comment with the attribute you want to discuss about, and reply in the thread. Otherwise please create a new comment for the attribute you want to discuss about.

To create an issue, please have a look at the [README](https://github.com/superlinked/VectorHub/blob/main/docs/tools/vdb_table/README.md) for contributing guidelines.

Let's all try to have a fruitful discussion!

**Comment by abid-redis** [comment]
Hi @AruneshSingh 
My recent [changes to the redis.json](https://github.com/superlinked/VectorHub/pull/589) file have been merged into main, but the [vector-db-comparison UI](https://superlinked.com/vector-db-comparison) isn’t showing the updates yet. Is there a refresh/update cycle for the vector-db-comparison table, or is there anything else I need to do on my end?
